src/vis/vis.js

```diff
@@ -262,29 +268,46 @@ var VisModel = Backbone.Model.extend({
     }
   },
 
+  init: function (options) {
```

Comment:
How are we planning to use this? `vis.instantiateMap` [can only be called once](https://github.com/CartoDB/cartodb.js/blob/a82b9b3e227099e41e454e97d677bf2a1a478cd9/src/vis/vis.js#L291). Do you think we have this logic there instead of adding a new public method?